skins/moono-lisa/dialog.css

```
@@ -752,6 +752,11 @@ select.cke_dialog_ui_input_select
 	margin-left: 1px;
 }
 
+/* (#3433) */
+.cke_dialog_ui_labeled_required{
+	color: red;
+}
```

Let's stick to the file styling.
```suggestion
/* (#3433) */
.cke_dialog_ui_labeled_required
{
	color: red;
}
```